packages/passport/src/firebase/authenticate-customer.ts

```diff
@@ -67,8 +67,10 @@ const authenticateCustomer = async (firebaseAuthToken: string) => {
         if (expires && new Date(expires).getTime() - Date.now() >= 2 * 60 * 1000) {
           return storedToken;
         }
-        const customerToken = generateAccessToken(customerId);
-        docRef.set(customerToken).catch(logger.error);
+        const customerToken = await generateAccessToken(customerId);
+        if (customerToken) {
+          docRef.set(customerToken).catch(logger.error);
+        }
```

Comment:
```suggestion
        docRef.set(customerToken).catch(logger.error);
```

Esse if é desnecessário, a função `generateAccessToken` retorna uma promise que resolve um objeto ou rejeita, em nenhuma hipótese `customerToken` vai ser nulo ou vazio, ou vai ser um objeto ou nem vai chegar nesse ponto (tem um throw error antes se a promise rejeitar)